iothub/device/tests/PlugAndPlay/PnpHelperTests.cs

```diff
@@ -0,0 +1,161 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using FluentAssertions;
+using Microsoft.Azure.Devices.Client.PlugAndPlay;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Azure.Devices.Client.Tests.PlugAndPlay
+{
+    [TestClass]
+    [TestCategory("Unit")]
+    public class PnpHelperTests
+    {
+        [TestMethod]
+        public void CreateRootPropertyPatch()
+        {
+            // Format:
+            //  {
+            //      "samplePropertyName": 20
+            //  }
+
+            string propertyName = "someName";
+            int propertyValue = 10;
+
+            string patch = PnpHelper.CreatePropertyPatch(propertyName, JsonConvert.SerializeObject(propertyValue));
+            var jObject = JObject.Parse(patch);
```

Comment:
JObject jObject as per coding guidelines.

Reply:
@vinagesh  I thought it was intuitive that `ABC.Parse()` is going to return `ABC`. If it isn't, let me know; I can specify the return type explicitly.

Reply:
Seems fine to me. It isn't as obvious as `var foo = new Foo()`, but still pretty obvious.